msu/hooks/config/items.nut

```diff
@@ -108,6 +108,56 @@ foreach (itemType in ::Const.Items.ItemType)
 	throw ::MSU.Exception.KeyNotFound(_weaponType);
 }
 
+// Map WeaponTypes to relevant strings from weapon.m.Categories.
+// Is used during automatic assignment of weapontypes to weapons.
+// Translators should push strings to the relevant weaponType here.
+// Key = WeaponType
+// Value = array of strings from weapon.m.Categories that should match to this weapon type
+
+// Translators Note: DO NOT OVERWRITE THIS TABLE, instead push your strings to each array e.g.
+// ::Const.Items.WeaponTypesCategoriesStrings.Axe.push("TranslatedAxe");
```

Comment:
This example won't work because key is not "Axe" here. Probably should be still Axe for consistency.